src/frontend/src/redux/search/reducer.ts

```diff
@@ -157,6 +163,154 @@ export function searchReducer(
         loading: action.ambiguous_charge_id,
       };
     }
+
+    case EDIT_CASE: {
+      const edits = JSON.parse(JSON.stringify(state.edits));
+      if (!edits[action.case_number]) {
+        edits[action.case_number] = {};
+      } // This check lets edits merge and not overwrite each other (e.g. AnswerDisposition vs EditCase)
```

Comment:
The idomatic pattern seems to be `edits[action.case_number] = edits[action.case_number] || {}`